### PR TITLE
Bookstore structure

### PR DESCRIPTION
**Pull request by algerina**
In this Milestone I did the following:
- Initialised React app.
- Added React Redux 
- Structured the application  files using a "feature folder" approach and using the ducks' pattern for  Redux files
- Added React Router
- Set the different components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```
Comment (GzimAsani) [inline package.json:5]: - [x] `react-redux` and should be present in `package.json`. Will be needed in order to use Redux in the next millstones/projects

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (GzimAsani) [inline README.md]: - [x] Please add a personalized readme file

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @algerina                               
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @algerina, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**